README.md

```diff
@@ -1,4 +1,76 @@
-# NEON-DC-DataLesson-Hackathon
-A repo for the spatio-temporal data lesson hackathon.
+# Welcome to the NEON Working with Vector Data in R Tutorial Series Git Repo!
+This repo is the building space for lessons on working with Vector Data in R.
+Once the lessons are developed and ready for publication, the lessons are moved
+to and published in the NEON Data Skills Git Repo and live at
+[www.neondataskills.org](http://www.neondataskills.org). 
+
+# NEON Science & Education
+NEON Data Skills provides tutorials and resources for working with scientific
+data. NEON 
+[(National Ecological Observatory Network](http://www.neonscience.org]) 
+is an ecological observatory, solely funded by the National Science Foundation 
+that will collect and provide open data for 30 years. 
+
+[www.neondataskills.org](http://www.neondataskills.org)
+
+Education is a key component of the NEON mission. As such this site serves to 
+facilitate use of data in both science and education.
+
+---
+
+### Work with Spatio-temporal Data in R -- Data Theme: Phenology
+The original materials were developed during a lesson building hack-a-thon 
+jointly organized by Leah Wasser (NEON), Tracy Teal (Data Carpentry), Mike 
+Smorul (National Socio-Environmental Synthesis Center (SESYNC) and Jason William
+(iPlant Collaborative). 
+
+Hack-a-thon participants included Mike Alonzo, Sean Barberie, Ben Best, 
+Zack Bryn, Jonah Duckles, Marisa Guarinello, Jeff Hollister, Megan A. Jones, 
+Matthew Kwit, Kristina Riemer, Dave Roberts, Keely Roth, Mike Smorul, 
+Courtney Soderberg, Joseph Stachelek, Kaitlin Stack Whitney, Tracy Teal, 
+Leah A. Wasser, Jason Williams, and Meg Williams.
+
+The results of this collaboration are four tutorial 
+series hosted on the NEON Data Skills portal (neondataskills.org) and also 
+taught as Data Carpentry workshops (www.datacarpentry.org).  
+
+* Introduction to Working With Spatio-Temporal Data and Data Management
+* Introduction to Working With Raster Data in R
+* Working With Raster Time Series Data in R
+* Introduction to Working With Time Series Data in Text Formats in R
+
+## Contributors
```

Comment:
we need to think about this language. Sarah is a reviewer whereas joe and you and i both actually developed the content. We should include a contact on this doc too 
Please send questions about these lessons to:
neondataskills@neoninc.org